.travis.yml

```diff
@@ -2,3 +2,9 @@ language: ruby
 rvm:
   - 2.1.0
   - 2.2.0
+  - 2.0.0
+  - 1.9.3
```

Comment:
I'm not sure if we convert this to an option hash, will this be a breaking change to existing callers?

If yes, I'd vote to not make a breaking change and remove 1.9 support.  If no, then refactor to an options hash seems right.